pkg/pillar/cmd/zedagent/attesttask.go

```diff
@@ -129,6 +131,29 @@ func (server *VerifierImpl) SendNonceRequest(ctx *zattest.Context) error {
 	return nil
 }
 
+//encodeVersions fetches fetches EVE, UEFI versions
+func encodeVersions(quoteMsg *attest.ZAttestQuote) error {
+	quoteMsg.Versions = make([]*attest.AttestVersionInfo, 0)
+	eveVersion := new(attest.AttestVersionInfo)
+	eveVersion.VersionType = attest.AttestVersionType_ATTEST_VERSION_TYPE_EVE
+	eveRelease, err := ioutil.ReadFile(types.EveVersionFile)
+	if err != nil {
+		return err
+	}
+	eveVersion.Version = string(eveRelease)
+
+	quoteMsg.Versions = append(quoteMsg.Versions, eveVersion)
+
+	uefiVersion := new(attest.AttestVersionInfo)
+	uefiVersion.VersionType = attest.AttestVersionType_ATTEST_VERSION_TYPE_FIRMWARE
+	biosVendor, biosVersion, biosReleaseDate := hardware.GetDeviceBios()
+	uefiVersion.Version = strings.TrimSpace(biosVendor) + "-" +
+		strings.TrimSpace(biosVersion) + "-" +
+		strings.TrimSpace(biosReleaseDate)
```

Comment:
Do we know whether '-' is used in the strings we combine?
Would make sense to check in zedcloud for the devices which we currently have.